backend/character/jobs.py

```diff
@@ -0,0 +1,130 @@
+''' Representation of static information at the class level '''
+
+from enum import Enum, auto
+import itertools
+
+class Roles(Enum):
+    ''' Roles enum '''
+    TANK = auto()
+    HEALER = auto()
+    MELEE = auto()
+    RANGED = auto()
+    CASTER = auto()
+
+
+class Buffs(Enum):
```

Comment:
i think keeping roles and jobs in this file is fine, but buffs seems like a separate thing and should go in another file

Reply:
I think I disagree here.  Buffs are ultimately a property solely bound to a job, no buff can belong to a job other than the one that is it.  Unless buffs take on substantial functionality of their own, I think the relation is enough to justify its presence in jobs.py